README.md

```diff
@@ -1,15 +1,30 @@
 # Microsoft IIS Web Manager
 Microsoft IIS Web Manager provides a browser interface for users to manage local and remote installations of IIS.
 
-### Requirements
-* The [Microsoft IIS Administration API](https://github.com/microsoft/iis.administration) must be installed on the target machine
+## Building it yourself
 
-### Building
-* [Typescript 2.1.* For Visual Studio 2015 ](https://www.microsoft.com/en-us/download/details.aspx?id=48593)
-* [Microsoft Visual Studio 2015](https://www.visualstudio.com/en-us/news/releasenotes/vs2015-update3-vs)
-* The project will build in Visual Studio directly after cloning
+#### Prerequisites
 
-### Images
+* [Microsoft Visual Studio 2017](https://docs.microsoft.com/en-us/visualstudio/releasenotes/vs2017-relnotes) or [Microsoft Visual Studio 2017 Build Tools](https://visualstudio.microsoft.com/thank-you-downloading-visual-studio/?sku=BuildTools&rel=15)
+
+* [Typescript 2.8.* For Visual Studio 2017](https://marketplace.visualstudio.com/items?itemName=TypeScriptTeam.typescript-281-vs2017)
+
+Run the following in developer command prompt for Visual Studio
+
+    msbuild Microsoft.IIS.WebManager.sln
```

Comment:
This failed for me. 

"c:\temp\IIS.WebManager\Microsoft.IIS.WebManager.sln" (default target) (1) ->
"c:\temp\IIS.WebManager\src\Microsoft.IIS.WebManager\Microsoft.IIS.WebManager.njsproj" (default target) (2) ->
(Version Stamping target) ->
  c:\temp\IIS.WebManager\src\build\versions.Targets(14,5): error MSB3073: The command "Powershell.exe c:\temp\IIS.WebMa
nager\src\Microsoft.IIS.WebManager\..\scripts\transform.ps1 -inputFile c:\temp\IIS.WebManager\src\Microsoft.IIS.WebMana
ger\index.html.versioned -symbol '$(IISWebManagerVersion)' -value '2.2.2' -outputFile c:\temp\IIS.WebManager\src\Micros
oft.IIS.WebManager\index.html" exited with code 1. [c:\temp\IIS.WebManager\src\Microsoft.IIS.WebManager\Microsoft.IIS.W
ebManager.njsproj]

Reply:
It's odd that when you run your build `$(MSBuildProjectDirectory)\..\scripts\transform.ps1` resolves to `c:\temp\IIS.WebMa
nager\src\Microsoft.IIS.WebManager..\scripts\transform.ps1`. Which msbuild version are you using